src/Utils/auth.ts

```diff
@@ -0,0 +1,19 @@
+import jwt from "jwt-decode";
+
+const decodeToken = (token: string) => {
+  const payloadToken: {
```

Comment:
Create that interface giving it a name so you can re-use it

Reply:
done thank you